LogicalTypes.md

```diff
@@ -228,7 +229,7 @@ integer. A precision too large for the underlying type (see below) is an error.
   warning
 * `fixed_len_byte_array`: precision is limited by the array size. Length `n`
   can store &lt;= `floor(log_10(2^(8*n - 1) - 1))` base-10 digits
-* `binary`: `precision` is not limited, but is required. The minimum number of
+* `byte_array`: `precision` is not limited, but is required. The minimum number of
```

Comment:
Perhaps we can change this to name the types using the same names as elswhere:

```suggestion
* `FIXED_LEN_BYTE_ARRAY `: precision is limited by the array size. Length `n`
  can store &lt;= `floor(log_10(2^(8*n - 1) - 1))` base-10 digits
* `BYTE_ARRAY `: `precision` is not limited, but is required. The minimum number of

Reply:
Yes, the formatting here is odd...all of the type names are lower case. Also, the use of back ticks for type names is inconsistent.  Perhaps (as you suggest) for this PR we can solely worry about binary->byte_array, and then do a second pass to fix capitalization and quoting.

Reply:
Looking at this more, it seems I'm introducing more inconsistency with the quoting. I'll try to clean that up.